request/mandatory_parameters.go

```diff
@@ -0,0 +1,18 @@
+package request
+
+import "net/http"
+
+//ErrorMissingRequiredParameters is returned when a request does not contain required parameters
+var ErrorMissingRequiredParameters error
```

Comment:
Yeah, either make the ValidParameters() method return an error if it fails, or just remove this error altogether.